rfc-009-transparent-code-bases.md

```diff
@@ -0,0 +1,18 @@
+# Transparent Codebases
+
+## Summary
+
+Teams are expected to publish details on accessing their source code.
+
+## Problem
+
+It is currently difficult for teams to see what other teams are working on, and gain insights, provide feedback and generally for us as a company to learn from each other.
+
+Even in the case where projects are hosted on madetech github, it's hard to filter through the large number of repositories.
+
+## Proposal
+
+Teams must publish, to a centralised location, links to details on how any Made Tech engineer can access their customer code.
+
+In situations where NDAs preclude providing transparent access to codebases, there must be a document that outlines this and reasons why. 
```

Comment:
What if it isn't even NDA reasoning, in cases of the customer just not wanting to add more than the necessary people to the company GH account

Reply:
Have included a potential proposal around this on line https://github.com/madetech/rfcs/pull/12/files#diff-96afec47c53e5e2879b8cd927d3a2cfcR20 but unsure how feasible this actually is

Reply:
(Other ideas welcomed)